### DISCUSSION TITLE
gh CLI publickey error

### DISCUSSION THREAD

**Discussion by SirYancy** [community/community#111143] [New to GitHub]
### Hi everyone,

I'm logged in using the gh CLI on a windows machine and I've uploaded my ssh key and everything should be working, but whenever I try to run the command to download student repos from an assignment, I get the error that my public key is incorrect and the thing just quits. So, what am I doing wrong?

**Comment by samus-aran** [comment]
Welcome to the GitHub Community, @SirYancy , we're happy you're here! You are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction. I've gone ahead and moved it for you. Good luck!

**Comment by queenofcorgis** [comment]
Hi there @SirYancy - would you mind sharing the exact error message you are receiving?
